etna/experimental/classification/feature_extraction/weasel.py

```diff
@@ -7,18 +7,25 @@
 
 import numpy as np
 from numpy.lib.stride_tricks import sliding_window_view
-from pyts.approximation import SymbolicFourierApproximation
-from pyts.transformation import WEASEL
 from scipy.sparse import coo_matrix
 from scipy.sparse import csr_matrix
 from scipy.sparse import hstack
 from sklearn.feature_extraction.text import CountVectorizer
 from sklearn.feature_selection import chi2
 from typing_extensions import Literal
 
+from etna import SETTINGS
 from etna.experimental.classification.feature_extraction.base import BaseTimeSeriesFeatureExtractor
 from etna.experimental.classification.utils import padd_single_series
 
+if SETTINGS.classification_required:
```

Comment:
Are we ok with this?

Reply:
Don't get how it works, if user don't have pyts, what happens when he try to instantiate WEASEL?

Reply:
I reworked importing logic. Probably, it will work better and more clear.